lib/internal/filter.js

```diff
@@ -1,19 +1,19 @@
-import filter from 'lodash/_arrayFilter';
+import filter from 'lodash/filter';
```

Comment:
It iterates on the `arr` argument, which is documented as a "collection" type. The test I wrote would fail when using `lodash/_arrayFilter`.